01 OperatorOverloading/OperatorOverloading.Host/Operation.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OperatorOverloading.Model;
+
+namespace OperatorOverloading.Host
+{
+    class Operation
+    {
+       static void Main(string[] args)
+        {
+           double money = 0.0;
+           string tempCurrency = null;
+
+           Money money1 = new Money();
+           Money money2 = new Money();
+           Money money3 = new Money();
+
+           /*obtaining values or first object*/
+           Console.WriteLine("Enter Amount for first object");
+           while (!double.TryParse(Console.ReadLine(), out money))
+           {
+               Console.WriteLine(" Enter valid entry for money");
+           }
+           Console.WriteLine("Enter Currency for first object");
+           tempCurrency = Console.ReadLine();
+
+           /*assigning values to object*/
+           money1.Amount = money;
+           money1.Currency = tempCurrency;
+
+           /*obtaining values or second object*/
+           Console.WriteLine("Enter Amount for second object ");
+           while (!double.TryParse(Console.ReadLine(), out money))
+           {
+               Console.WriteLine(" Enter valid entry for money");
+           }
+           Console.WriteLine("Enter Currency for second object");
+           tempCurrency = Console.ReadLine();
+           
+           money2.Amount = money;
+           money2.Currency = tempCurrency;
+
+           try
+           {
+               money3 = money1 + money2;
+           }
+           catch (InvalidCurrencyException ex) // catched exception
```

Comment:
If you are going to print the Exception message every time, catch only System.Exception.